cms/djangoapps/course_to_library_import/api/v0/views.py

```diff
@@ -0,0 +1,40 @@
+"""
+API v0 views.
+"""
+
+from rest_framework.permissions import IsAdminUser
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from cms.djangoapps.course_to_library_import.py_api import import_library_from_staged_content
+from .serializers import ImportBlocksSerializer
+
+
+class ImportBlocksView(APIView):
+    """
+    Import blocks from a course to a library.
+    """
+
+    permission_classes = (IsAdminUser,)
+    serializer_class = ImportBlocksSerializer
+
+    def post(self, request, *args, **kwargs):
+        """
+        Import blocks from a course to a library.
+        """
+        data = self.serializer_class(data=request.data)
+        data.is_valid(raise_exception=True)
+
+        try:
+            import_library_from_staged_content(
```

Comment:
Do we have to check for a `CourseToLibraryImport` with a received UUID and "READY" status before starting the import?